NSelene/SeleneElement.cs

```diff
@@ -366,5 +382,30 @@ public static SeleneCollection SS(this SeleneElement selement, string cssSelecto
                 return selement.FindAll(cssSelector);
             }
         }
+
+        public static class JSElementExtensions
+        {
+            public static SeleneElement JSScrollIntoView(this SeleneElement selement)
```

Comment:
according to CamelCase rules, it should be named JsScrollIntoView

Reply:
let's also follow the following guideline... 
call element all SeleneElement instances, and call webelement all IWebElement instances...

Reply:
I know by the way that all over the code I wrote it as selement too) But let's then move to the new style:) selement even sounds a bit awkward:)

Reply:
regarding Js vs JS, I don't agree, and in the whole code base JS is used.

Reply:
All over the whole code base of nselene, it's just in one place - Have.JSReturnedTrue :)

![image](https://user-images.githubusercontent.com/1961118/86005399-46a11080-ba1d-11ea-961f-d078cf483acd.png)


and in another it's already like `Js`, see in `Configuration.setValueByJs`
 
Actually, I though that the "standard naming convention" strictly recommends to name it like Js, not JS, but it happened that for "two-symbols-abreviations" there is an exception, and actually it's fine to name it like JSClick...
see:
https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md#7-do-use-camelcasing-for-abbreviations-3-characters-or-more-2-chars-are-both-uppercase

Hm... For my taste the `JSClick` is a bit harder readable, because it's harder to see where the next word starts, it also like "too screaming" too much of "loudness" in so much of uppercase letters. 

I tend to choose:

```
JsClick
JsType
JsSetValue
JsScrollIntoView
```

vs

```
JSClick
JSType
JSSetValue
JSScrollIntoView
```
Look, how in the last examples these double-s makes it kind of "fuzzy"... not clean...

I recommend to stick to `JsStyle` ... and Have.JSReturnedTrue, keep as it is... Later we will add `Have.JsReturned(bool result, string script)` that will be more universal and powerful, and name it already according to the new style to follow...